main.py

```diff
@@ -222,17 +222,17 @@ def ui_access_wrapper(self, *args, **kwargs):
         r'/admintopicscsvdownloadhandler',
         admin.AdminTopicsCsvFileDownloader),
     get_redirect_route(
-        r'/addcontributionreviewerhandler',
-        admin.AddContributionReviewerHandler),
+        r'/addcontributionrightshandler',
```

Comment:
generally, we should avoid changing endpoints/URL but considering these are not user-facing URLs and the oppia APIs are not meant for public use, so it's fine changing things. (Adding this here as a reviewer for the main.py file and for future reference for the approval of this change.)